### PR TITLE
Use SLW component on UserProfile.

### PR DESCRIPTION
**Pull request by kadams54**

### CODE REVIEW

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5608/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5609/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5611/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5621/
Test PASSed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5622/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5624/
Test PASSed.

File: jujugui/static/gui/src/app/components/user-profile/agreement-list/agreement-list.js [hunk 1]
```diff
@@ -30,6 +31,7 @@ YUI.add('user-profile-agreement-list', function() {
       this.xhrs = [];
 
       return {
+        broadcastStatus: function() {},
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/user-profile/agreement-list/agreement-list.js]: I don't think you meant to leave this in.

File: jujugui/static/gui/src/app/components/user-profile/budget-list/budget-list.js [hunk 2]
```diff
@@ -30,6 +31,7 @@ YUI.add('user-profile-budget-list', function() {
       this.xhrs = [];
 
       return {
+        broadcastStatus: function() {},
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/user-profile/budget-list/budget-list.js]: left in

File: jujugui/static/gui/src/app/components/user-profile/entity-list/entity-list.js [hunk 3]
```diff
@@ -34,6 +35,7 @@ YUI.add('user-profile-entity-list', function() {
       this.xhrs = [];
 
       return {
+        broadcastStatus: function() {},
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/user-profile/entity-list/entity-list.js]: Left in

File: jujugui/static/gui/src/app/components/user-profile/model-list/model-list.js [hunk 4]
```diff
@@ -39,6 +40,7 @@ YUI.add('user-profile-model-list', function() {
       this.xhrs = [];
 
       return {
+        broadcastStatus: function() {},
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/user-profile/model-list/model-list.js]: Left in

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5635/
Test PASSed.

File: jujugui/static/gui/src/app/components/user-profile/agreement-list/agreement-list.js [hunk 5]
```diff
@@ -35,6 +36,12 @@ YUI.add('user-profile-agreement-list', function() {
       };
     },
 
+    getDefaultProps: function() {
+      return {
+        broadcastStatus: function() {}
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/user-profile/agreement-list/agreement-list.js]: Could you add a comment to all these as to why it's here

File: jujugui/static/gui/src/app/components/user-profile/agreement-list/agreement-list.js [hunk 6]
```diff
@@ -22,6 +22,7 @@ YUI.add('user-profile-agreement-list', function() {
 
   juju.components.UserProfileAgreementList = React.createClass({
     propTypes: {
+      broadcastStatus: React.PropTypes.func,
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/user-profile/agreement-list/agreement-list.js]: Could you add a comment as to where this is coming from.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5636/
Test PASSed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5637/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5638/
Test PASSed.

**Comment by hatched** [comment]
After discussing with @kadams54 - This branch is not properly qa'able because of errors elsewhere in the stack. This branch will land as-is without QA and once the branch which builds on this one to resolve the errors lands a thorough QA will be necessary.

:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**